### PR TITLE
Turkish translation of the PHP 8 Announcement Page

### PR DESCRIPTION
**Pull request by emir**
We translated PHP 8 announcement page to Turkish with @aliozgurr✅

### CODE REVIEW

**Comment by PeeHaa** [comment]
Will ask @ekinhbayar to review tomorrow 👍

**Comment by emir** [comment]
Resolved #368 conflict.

File: releases/8.0/tr.php [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+            </div>
+        </div>
+    </section>
+
+    <section class="php8-section php8-section_light">
+        <h2 class="php8-h2">Just-In-Time Derlemesi (JIT)</h2>
+        <p>
+            PHP 8, iki JIT derleme motoru sunuyor. Tracing JIT, ikisi arasında en yetenekli olanı. Karşılaştırmalarda
+            yaklaşık 3 kat daha iyi performans ve uzun süre işlem yapan bazı uygulamalarda 1,5–2 kat iyileşme gösteriyor.
+            Normal uygulamalarda performansı PHP 7.4 ile aynı.</p>
```
Comment (ekinhbayar) [inline releases/8.0/tr.php:367]: Let's remove the line break here?

File: releases/8.0/tr.php [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+                        Değişken atamasına gerek olmayan hataların yakalanabilmesi <a href="https://wiki.php.net/rfc/non-capturing_catches">RFC</a>
+                    </li>
+                    <li>
+                        Değişken sözdizimlerinde iyileştirmeler <a href="https://wiki.php.net/rfc/variable_syntax_tweaks">RFC</a>
+                    </li>
+                    <li>
+                        İsim alanındaki tanımları tek bir belirteç olarak değerlendirme <a href="https://wiki.php.net/rfc/namespaced_names_as_token">RFC</a>
+                    </li>
+                    <li>
+                        Throw deyimi artık bir ifade(expression) <a href="https://wiki.php.net/rfc/throw_expression">RFC</a>
```
Comment (ekinhbayar) [inline releases/8.0/tr.php:440]: space before (expression)

**Review by ekinhbayar — COMMENTED** [review]
Thanks for working on this! Besides some suggestions, the main change needed is correcting the indentation to keep it consistent with other translations.

File: releases/8.0/tr.php [hunk 3]
```diff
@@ -0,0 +1,503 @@
...
+                    <div class="php8-code phpcode">
+                        <?php highlight_php_trimmed(
+                            'htmlspecialchars($string, double_encode: false);'
+                        );?>
+                    </div>
+                </div>
+            </div>
+            <div class="php8-compare__content">
+                <ul>
+                    <li>İsteğe bağlı parametreler atlanabiliyor ve yalnızca gerekli olanlar belirtilebiliyor.</li>
```
[the remark this answers is not in the document]
Comment (emir) [reply releases/8.0/tr.php:66]: Cümlenin sonunu **belirtebiliyorsunuz** olarak değiştirdim. Çevirinin geri kalanında da _kullanılabiliyor_, _yapılabiliyor_ gibi ele aldığım için tutarlığı bozmuyorum.

File: releases/8.0/tr.php [hunk 4]
```diff
@@ -0,0 +1,503 @@
...
+            </div>
+            <div class="php8-compare__content">
+                <p>Değişken türlerinin kombinasyonu için PHPDoc açıklamaları yerine çalışma zamanında doğrulanan
+                    birleşim türleri(union type) tanımlamaları kullanılabiliyor.</p>
+            </div>
+        </div>
+
+        <div class="php8-compare">
+            <h2 class="php8-h2" id="match-expression">
+                Match Deyimi
```
[the remark this answers is not in the document]
Comment (emir) [reply releases/8.0/tr.php:210]: Done. Also Updated "Deyimi" I used at all the other parts of translation👍

File: releases/8.0/tr.php [hunk 5]
```diff
@@ -0,0 +1,503 @@
...
+                        );?>
+                    </div>
+                </div>
+            </div>
+            <div class="php8-compare__content">
+                <p>Yeni match deyimi switch'e çok benzer ve aşağıdaki özelliklere sahiptir:</p>
+                <ul>
+                    <li>Match bir ifadedir, sonucu bir değişkende saklanabilir veya döndürülebilir.</li>
+                    <li>Match'in karşılıkları tek satır ifadeleri destekler ve break; kullanılması gerekmez.</li>
+                    <li>Match katı tip(strict) karşılaştırma yapar.</li>
```
[the remark this answers is not in the document]
Comment (emir) [reply releases/8.0/tr.php:250]: Done.

File: releases/8.0/tr.php (hunk 2 above)
Comment (emir) [reply releases/8.0/tr.php:440]: Done.

File: releases/8.0/tr.php (hunk 1 above)
Comment (emir) [reply releases/8.0/tr.php:367]: No, you're right. There was an extra closing p tag there. Done.

File: releases/8.0/tr.php [hunk 6]
```diff
@@ -0,0 +1,503 @@
...
+            </div>
+            <div class="php8-compare__content">
+                <p>Sayısal string karşılaştırılırken PHP 8 sayısal olarak karşılaştırır. Aksi halde sayı bir
+                    string'e çevrilir ve string olarak karşılaştırılır.</p>
+            </div>
+        </div>
+
+        <div class="php8-compare">
+            <h2 class="php8-h2" id="consistent-type-errors-for-internal-functions">
+                Dahili Fonksiyonlar için Tutarlı Hata Türleri
```
[the remark this answers is not in the document]
Comment (emir) [reply releases/8.0/tr.php:330]: Done. I don't like it but keeping consistency with the official translation, I updated.

File: releases/8.0/tr.php [hunk 7]
```diff
@@ -0,0 +1,503 @@
...
+        <div class="page-tools">
+            <div class="change-language">
+                <?php releases\php80\language_chooser('tr'); ?>
+            </div>
+        </div>
+        <div class="php8-section__content">
+            <div class="php8-logo">
+                <img src="/images/php8/logo_php8.svg" alt="php8" height="126" width="343">
+            </div>
+            <div class="php8-title">Yayında!</div>
```
[the remark this answers is not in the document]
Comment (emir) [reply releases/8.0/tr.php:23]: "Yayında!" looks stronger to me😎

File: releases/8.0/tr.php [hunk 8]
```diff
@@ -0,0 +1,503 @@
...
+{
+    #[Route("/api/posts/{id}", methods: ["GET"])]
+    public function get($id) { /* ... */ }
+}'
+                        );?>
+                    </div>
+                </div>
+            </div>
+            <div class="php8-compare__content">
+                <p>PHPDoc yorum satırları yerine PHP sözdizimi ile yapılandırılmış metadata kullanılabiliyor.</p>
```
[the remark this answers is not in the document]
Comment (emir) [reply releases/8.0/tr.php:107]: I leave it "kullanılabiliyor" for keeping consistent with other parts of the translation.

File: releases/8.0/tr.php [hunk 9]
```diff
@@ -0,0 +1,503 @@
...
+}
+
+new Number(\'NaN\'); // TypeError'
+                        );?>
+                    </div>
+                </div>
+            </div>
+            <div class="php8-compare__content">
+                <p>Değişken türlerinin kombinasyonu için PHPDoc açıklamaları yerine çalışma zamanında doğrulanan
+                    birleşim türleri(union type) tanımlamaları kullanılabiliyor.</p>
```
[the remark this answers is not in the document]
Comment (emir) [reply releases/8.0/tr.php]: 1st Done.
2nd: I leave it "kullanabilirsiniz".

File: releases/8.0/tr.php [hunk 10]
```diff
@@ -0,0 +1,503 @@
...
+
+releases\php80\language_redirect('tr');
+
+releases\php80\common_header(
+    'PHP 8.0, PHP dili için önemli bir güncellemedir. Optimizasyonlar ve yeni özellikler: Adlandırılmış ' .
+    'Değişkenler, Union Types, Attributes, Kurucularda Özellik Tanımı, Match Deyimi, Nullsafe Operatorü, ' .
+    'JIT(Anında Derleme) yanında tip sistemi, hata işleme ve tutarlılıkta iyileştirmeler içerir.');
+
+?>
+    <section class="php8-section php8-section_dark php8-section_header center">
```
[the remark this answers is not in the document]
Comment (emir) [reply releases/8.0/tr.php]: Done.

**Comment by emir** [comment]
All done, here🤙

**Comment by ekinhbayar** [comment]
@cmb69 we're good to go here :-)

**Comment by cmb69** [comment]
Thank you both!

**Pull request closed without merging**